### PR TITLE
Adding Chinese-Simplified localizations.

### PR DESCRIPTION
**Pull request by megabitsenmzq**
Full translation for Chinese-Simplified.
Also, fixed a small problem in other strings.

### CODE REVIEW

File: Xcodes/Resources/zh-Hans.lproj/Localizable.strings [hunk 1]
```diff
@@ -0,0 +1,218 @@
...
+// Common
+"Install" = "安装";
+"InstallDescription" = "安装此版本";
+"RevealInFinder" = "在访达中显示";
+"Active" = "已激活";
+"MakeActive" = "激活";
+"Open" = "打开";
+"OpenDescription" = "打开此版本";
+"CopyPath" = "复制文件位置";
+"CreateSymLink" = "以 Xcode.app 创建短链接";
```
Comment (iXerol) [inline Xcodes/Resources/zh-Hans.lproj/Localizable.strings:18]: 「短链接」不恰当，常用翻译应为「符号链接」「软链接」（前者更规范）

File: Xcodes/Resources/zh-Hans.lproj/Localizable.strings [hunk 2]
```diff
@@ -0,0 +1,218 @@
+// Menu
+"Menu.About" = "关于 Xcodes";
+"Menu.CheckForUpdates" = "检查更新…";
+"Menu.Acknowledgements" = "Xcodes 致谢";
+"Menu.GitHubRepo" = "Xcodes GitHub Repo";
```
Comment (iXerol) [inline Xcodes/Resources/zh-Hans.lproj/Localizable.strings:5]: 与下文中这句不统一。
`"GithubRepo" = "GitHub 仓库";`

File: Xcodes/Resources/zh-Hans.lproj/Localizable.strings [hunk 3]
```diff
@@ -0,0 +1,218 @@
...
+"VersionWithBuild" = "版本 %@ (%@)";
+"GithubRepo" = "GitHub 仓库";
+"Acknowledgements" = "致谢";
+"UnxipExperiment" = "Unxip Experiment";
+"License" = "许可协议";
+
+// General Preference Pane
+"General" = "通用";
+"AppleID" = "Apple ID";
+"SignIn" = "登陆";
```
Comment (iXerol) [inline Xcodes/Resources/zh-Hans.lproj/Localizable.strings:60]: 「登录」，多处相同问题

**Review by iXerol — CHANGES_REQUESTED** [review]

File: Xcodes/Resources/zh-Hans.lproj/Localizable.strings (hunk 1 above)
Comment (megabitsenmzq) [reply Xcodes/Resources/zh-Hans.lproj/Localizable.strings]: 这个是我打错了，一会改一下。

**Comment by yuzhouwww** [comment]
我正准备提 PR。。。😭

File: Xcodes/Resources/en.lproj/Localizable.strings [hunk 4]
```diff
@@ -77,7 +77,7 @@
 "LocalCachePathDescription" = "Xcodes caches available Xcode versions and temporary downloads new versions to a directory";
 "Change" = "Change";
 "Active/Select" = "Active/Select";
-"AutomaticallyCreateSymbolicLink" = "Automatically create symbolic link to Xcodes.app";
+"AutomaticallyCreateSymbolicLink" = "Automatically create symbolic link to Xcode.app";
```
Comment (yuzhouwww) [inline Xcodes/Resources/en.lproj/Localizable.strings:80]: you can also fix this typo in other localizations

File: Xcodes/Resources/zh-Hans.lproj/Localizable.strings [hunk 5]
```diff
@@ -0,0 +1,218 @@
...
+"MacOSRequirement" = "需要macOS %@及以上";
+"SDKs" = "SDK";
+"Compilers" = "编译器";
+"DownloadSize" = "下载文件大小";
+"NoXcodeSelected" = "没有选定的Xcode版本";
+
+// Installation Steps
+// When localizing. Items will be replaced in order. ie "Step 1 of 6: Downloading"
+// So if changing order, make sure the positional specficier is retained. ex: "%3$@: Step %1$d of %2$d"
+"InstallationStepDescription" = "第%1$d步，共%2$d步：%3$@";
```
Comment (yuzhouwww) [inline Xcodes/Resources/zh-Hans.lproj/Localizable.strings:45]: I prefer "进度 1/6: 下载中", more brief compared to "第 1 步，共 6 步: 下载中"

File: Xcodes/Resources/zh-Hans.lproj/Localizable.strings [hunk 6]
```diff
@@ -0,0 +1,218 @@
...
+"DownloadingPercentDescription" = "正在下载：已完成%d%%";
```
Comment (yuzhouwww) [inline Xcodes/Resources/zh-Hans.lproj/Localizable.strings:46]: "已完成" is unnecessary here

File: Xcodes/Resources/zh-Hans.lproj/Localizable.strings (hunk 5 above)
Comment (megabitsenmzq) [reply Xcodes/Resources/zh-Hans.lproj/Localizable.strings:45]: This changes the sentence too much. I always want to keep the structure from the source language as I can.

File: Xcodes/Resources/en.lproj/Localizable.strings (hunk 4 above)
Comment (megabitsenmzq) [reply Xcodes/Resources/en.lproj/Localizable.strings:80]: I will check it in other languages when I have time this week.

File: Xcodes/Resources/zh-Hans.lproj/Localizable.strings (hunk 6 above)
Comment (megabitsenmzq) [reply Xcodes/Resources/zh-Hans.lproj/Localizable.strings:46]: As I mentioned above, I want to keep the structure from the source language as I can. Also, if "已完成" is unnecessary, the same words in English strings will be unnecessary too. So I kinda want to keep this.

**Comment by MattKiazyk** [comment]
I've closed  #228 as a duplicate of this PR but @hdcola and @Sichengthebest if you have any comments on @megabitsenmzq implementation, please feel free to make a comment.

**Comment by MattKiazyk** [comment]
@megabitsenmzq is this ready for review? 

There's a conflict because we've had a few new languages all updating the same file. If you could fix that up, or let me know it's ready and I can fix up.

**Comment by MattKiazyk** [comment]
Thanks @megabitsenmzq and all!

**Pull request merged**